### DISCUSSION TITLE
target server without zrepl

### DISCUSSION THREAD

**Discussion by dionysius** [zrepl/zrepl#667] [Q&A]
A) Is it possible to have zrepl only on the source side installed and configured? It could gather all info via ssh+zfs commands, do the diff internally and then decides what and how to push with plain zfs commands.

B) Is it possible to have sink configuration only on the source side for the target side if the sink side can never be (re)configured? (Both sides have zrepl installed)

Use case: Friend to Friend offsite-backup. I want to coordinate replication to an untrusted and not from me managed zfs server, I don't want to give the owner the burden to (re)configure the sink (and depending on how we setup restrictions we won't be able to - keywords: ssh forced command, separate zfs user with own permission).

The talk slide was promising but the deeper I dig the more I question doability:
![screenshot 248](https://user-images.githubusercontent.com/1341084/211071258-79471f2b-7477-4bad-b22c-941a3746738a.jpg)

**Comment by Malvineous** [comment]
I had a similar use case - I wanted to replicate a ZFS dataset onto another machine, but I didn't want to have to install a ZFS-enabled Linux kernel on it.

I ended up setting up iSCSI on both machines and exporting the disk that way, so that I could mount the remote disk on the local ZFS host.  At that point from zrepl's perspective it just became a local replication job between two disks on the same machine.
